lib/src/internal/http_endpoints.dart

```diff
@@ -556,6 +559,25 @@ class HttpEndpoints implements IHttpEndpoints {
     return Future.error(response);
   }
 
+  @override
+  Future<IUser> fetchEmojiCreator(Snowflake guildId, Snowflake emojiId) async {
+    final response = await httpHandler.execute(BasicRequest("/guilds/$guildId/emojis/$emojiId"));
+
+    if (response is HttpResponseSuccess) {
+      if(response.jsonBody["managed"] as bool) {
+        return Future.error("Emoji is managed");
```

Comment:
can you wrap returned error string with ArgumentError class. Thats the pattern in whole file. Its not ideal, but Im gonna deal with it in the future